### PR TITLE
Added Azure AAD integration

### PR DESCRIPTION
**Pull request by yashints**
Added AAD integration, need to add some tests

### CODE REVIEW

File: src/AAD/getUserProfile.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    headers.append('Authorization', bearer);
+    var options = {
+      method: 'GET',
+      headers: headers
+    };
+    var graphEndpoint = 'https://graph.microsoft.com/v1.0/me';
+    return fetch(graphEndpoint, options)
+      .catch(error => {
+        console.error(
+          'Failed to get your profile, you have ti fill the form manually'
```
Comment (xwipeoutx) [inline src/AAD/getUserProfile.js]: Typo! ~to~ to

Comment (yashints) [reply src/AAD/getUserProfile.js]: fixed

**Comment by RobFaie** [comment]
AzureAD side looks fine. I did an admin grant for Readify and can do one for Kloud once this gets deployed.

Note that we haven't yet been able to get multi-tenant working with Telstra. We are making progress, but don't expect things to work for them day 1.

**Comment by yashints** [comment]
We won't need admin consent since the app is using delegated scope. It would make user's job easier but not necessary

**Pull request merged**